src/systems/kinetic_energy_monitor/KineticEnergyMonitor.cc

```diff
@@ -127,40 +127,13 @@ void KineticEnergyMonitor::Configure(const Entity &_entity,
   transport::Node node;
   this->dataPtr->pub = node.Advertise<msgs::Double>(topic);
 
-  if (!_ecm.Component<components::WorldPose>(this->dataPtr->linkEntity))
-  {
-    _ecm.CreateComponent(this->dataPtr->linkEntity,
-        components::WorldPose());
-  }
+  Link link(this->dataPtr->linkEntity);
+  link.EnableVelocityChecks(_ecm, true);
 
   if (!_ecm.Component<components::Inertial>(this->dataPtr->linkEntity))
```

Comment:
This is one such place where the templated command would be very powerful.

Reply:
Using `enableComponent` here now: 9f61ff885ad48bc7c010a9ef964e2fbf31f2d062